etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+
```

Comment:
```suggestion
```

Reply:
Why remove this header comment?

Reply:
No other profile has a empty line there.

Reply:
Oh, right.  Sorry, I completely misread the diff.